.goreleaser.yml

```diff
@@ -0,0 +1,35 @@
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - make clean
+    - env VERSION="{{ .Version }}" make build
+builds:
+  - env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -X github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/cmd.Version={{ .Version }}
+    goos:
+      - linux
+      - windows
+      - darwin
+    binary: relok8s
+archives:
+  - replacements:
+      darwin: Darwin
```

Comment:
this replacements to add capitalization does not feel right, in fact I do not like the filename resulting of it which uses underscore + Capitalized at the same time.

To me the only two replacements here should be the ones related to the arch

Reply:
done